### DISCUSSION TITLE
Rom Selection by letter

### DISCUSSION THREAD

**Discussion by billylepandaroux** [OnionUI/Onion#975] [Ideas]
Hi

Just 2 ideas to select a rom
I know, there's a search engine but i think its gonna be helpful too to have these other options to improve user's experience

1 - Mostly any music app on smartphone for example has this kind of menu (on the right here) 
![Poweramp](https://github.com/OnionUI/Onion/assets/56294861/df06fb04-c84d-4544-b7ff-4a7a082f3fab)

And / Or 

2 - with R2/L2, jump to next/previous letter (like on Recalbox)

**Comment by Aemiii91** [comment]
Definitely, unfortunately the frontend we're currently using is closed source - meaning, we can't add this kind of stuff.

**Comment by billylepandaroux** [comment]
Thanx for the fast answer.
Too bad that you cant.
